src/Uno.UI/UI/Xaml/Controls/Grid/RowDefinition.cs

```diff
@@ -51,8 +51,6 @@ public double MinHeight
 			set => SetMinHeightValue(value);
 		}
 
-		private static GridLength GetMaxHeightDefaultValue() => GridLengthHelper.OneStar;
-
```

Comment:
Nice catch

Reply:
This was caught by the fix for https://github.com/unoplatform/uno/issues/9940 🎉